### PR TITLE
fix(stax): update onboarding flow (firmware update)

### PR DESCRIPTION
**Pull request by ak-ledger**
### 📝 Description

Review the firmware update steps when onboarding a new Stax device or updating firmware on any device.

https://github.com/LedgerHQ/ledger-live/assets/146743014/dd23df66-5628-49b7-803c-8c0fd451cfc3

### ❓ Context

LLD
[JIRA-9990](https://ledgerhq.atlassian.net/browse/LIVE-9990?filter=13654)

### ✅ Checklist

Pull Requests must pass the CI and be code reviewed. Set as Draft if the PR is not ready.

- [x] `npx changeset` was attached.
- [ ] **Covered by automatic tests.** 
- [ ] **Impact of the changes:** 
  - 

---

### 🧐 Checklist for the PR Reviewers

- [ ] **The code aligns with the requirements** described in the linked JIRA or GitHub issue.
- [ ] **The PR description clearly documents the changes** made and explains any technical trade-offs or design decisions.
- [ ] **There are no undocumented trade-offs**, technical debt, or maintainability issues.
- [ ] **The PR has been tested** thoroughly, and any potential edge cases have been considered and handled.
- [ ] **Any new dependencies** have been justified and documented.
- [ ] **Performance** considerations have been taken into account. (changes have been profiled or benchmarked if necessary)

### CODE REVIEW

**Comment by KVNLS** [comment]
Actual
image
Expected
Screenshot 2023-11-23 at 15 36 57

Actual
Screenshot 2023-11-23 at 15 39 37
Expected
image


Actual
image
Expected
image

File: apps/ledger-live-desktop/src/renderer/components/SyncOnboarding/Manual/SyncOnboardingCompanion.tsx [hunk 1]
```diff
@@ -160,6 +161,7 @@ const SyncOnboardingCompanion: React.FC<SyncOnboardingCompanionProps> = ({
         renderBody: () => (
           <Flex flexDirection="column">
             <TrackPage category={`Set up ${productName}: Step 2 PIN`} flow={analyticsFlowName} />
+            {/* @ts-expect-error weird props issue with React 18 */}
```
Comment (mcayuelas-ledger) [inline apps/ledger-live-desktop/src/renderer/components/SyncOnboarding/Manual/SyncOnboardingCompanion.tsx:164]: What is the error? :)

Comment (ak-ledger) [reply apps/ledger-live-desktop/src/renderer/components/SyncOnboarding/Manual/SyncOnboardingCompanion.tsx:164]: Don't remember exactly. Seems like the StepText is not typed correctly with styled-components.

Comment (mcayuelas-ledger) [reply apps/ledger-live-desktop/src/renderer/components/SyncOnboarding/Manual/SyncOnboardingCompanion.tsx:164]: Ok, i saw that is what we are doing currentlty so LGTM :) 

But we need to tackle that and fix it when we have a moment

File: apps/ledger-live-desktop/src/renderer/components/SyncOnboarding/Manual/EarlySecurityChecks/index.tsx [hunk 2]
```diff
@@ -170,7 +170,7 @@ const EarlySecurityChecks = ({
         closeFwUpdateDrawer();
         restartChecksAfterUpdate();
       },
-      deviceHasPin: false, // early security checks are triggered only if the device is in one of the steps prior to setting a PIN code
+      deviceHasPin: deviceModelId !== "stax", // early security checks are triggered only if the device is in one of the steps prior to setting a PIN code
```
Comment (mcayuelas-ledger) [inline apps/ledger-live-desktop/src/renderer/components/SyncOnboarding/Manual/EarlySecurityChecks/index.tsx:173]: I think there is a enum fort that no? `stax`

**Review by mcayuelas-ledger — APPROVED** [review]

**Pull request merged**